ArmPlatformPkg/ArmPlatformPkg.dsc

```diff
@@ -123,7 +123,6 @@
   ArmPlatformPkg/PrePeiCore/PrePeiCoreMPCore.inf
   ArmPlatformPkg/PrePeiCore/PrePeiCoreUniCore.inf
 
-  ArmPlatformPkg/PrePi/PeiMPCore.inf

```

Comment:
Comment actually on commit message (thanks github):
I think the word "from" or suchlike has been dropped before "[mostly]".